backend/benefit/helsinkibenefit/settings.py

```diff
@@ -274,6 +275,9 @@
 AZURE_ACCOUNT_NAME = env("AZURE_ACCOUNT_NAME")
 AZURE_ACCOUNT_KEY = env("AZURE_ACCOUNT_KEY")
 AZURE_CONTAINER = env("AZURE_CONTAINER")
+AZURE_URL_EXPIRATION_SECS = env("AZURE_URL_EXPIRATION_SECS")  # default 900s
+
+MAX_UPLOAD_SIZE = 13107200  # 10MB
```

Comment:
13107200 / (1024*1024) == 12.5?

Reply:
Didn't remember why I came up with this number 😅 Yes it should be 10485760